### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- new forms section
- new forms sidebar
- initial preparation for forms docs
- changed the docs to DevKit URL in the top bar 
- moved additional libraries as top-level category

# Screenshots / Videos
Screenshot 2023-07-13 at 09 24 59
Screenshot 2023-07-13 at 09 25 04

### CODE REVIEW

File: website/docs/additional-libraries/boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+id: boilerplate
+title: Eightshift Boilerplate
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-boilerplate)
+
+All the tools you need to start building a modern WordPress project using the latest front-end development tools.
```
Comment (goranalkovic-infinum) [inline website/docs/additional-libraries/boilerplate.md:8]:
```suggestion
All the tools you need to start building a modern WordPress project using the latest front end development tools.
```
I _do_ hate it with the space in-between, but looks like that's the correct way to write it 🥲

File: website/docs/additional-libraries/forms.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+id: forms
+title: Eightshift Forms
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-forms.svg?style=for-the-badge)](https://github.com/infinum/eightshift-forms)
+
+This plugin is made on our Eightshift Boilerplate. It lets you create a cool-looking form using our custom block and much more. We implemented a bunch of integrations and all sorts of cool stuff.
```
Comment (piqusy) [inline website/docs/additional-libraries/forms.md:8]:
```suggestion
This plugin is based on Eightshift Boilerplate. It lets you create a cool-looking forms using custom blocks and much more. We implemented a bunch of integrations and all sorts of cool stuff.
```

File: website/docs/additional-libraries/coding-standards.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+id: coding-standards
+title: Eightshift Coding Standards
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-coding-standards.svg?style=for-the-badge)](https://github.com/infinum/eightshift-coding-standards)
+
+This package contains [Eightshift Coding Standards for WordPress](https://handbook.infinum.co/books/wordpress) for [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/). The package intends to have a unified code across the WordPress projects we do at Eightshift and help with the code review.
```
Comment (piqusy) [inline website/docs/additional-libraries/coding-standards.md:8]:
```suggestion
This package contains [Eightshift Coding Standards for WordPress](https://handbook.infinum.co/books/wordpress) for [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/). The goal is to have a unified code across the WordPress projects we do at Eightshift and help with the code review.
```

File: website/docs/additional-libraries/storybook.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+id: storybook
+title: Eightshift Storybook
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-storybook.svg?style=for-the-badge)](https://github.com/infinum/eightshift-storybook)
+
+This library hosts all the packages necessary for loading Storybook in your project. We are making updates as a new core version comes out.
```
Comment (piqusy) [inline website/docs/additional-libraries/storybook.md:8]:
```suggestion
This library hosts all the packages necessary for loading Storybook in your project. We are making updates as a new core version come out.
```

File: website/forms/intro.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+id: intro
+title: Intro
+---
+
+Eightshift forms plugins is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```
Comment (iobrado) [inline website/forms/intro.md:6]:
```suggestion
Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

**Review by iobrado — APPROVED** [review]

File: website/forms/intro.md (hunk 5 above)
Comment (piqusy) [inline website/forms/intro.md]:
```suggestion
Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

File: website/forms/intro.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+id: intro
+title: Intro
+---
+
+Eightshift forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```
Comment (goranalkovic-infinum) [inline website/forms/intro.md:6]:
```suggestion
Eightshift Forms is a full-featured form builder plugin that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice start 👏🏼 
Left some wording suggestions

File: website/forms/integrations/mailerlite.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+---
+id: mailerlite
+title: Mailerlite
```
Comment (goranalkovic-infinum) [inline website/forms/integrations/mailerlite.md:3]:
```suggestion
title: MailerLite
```

File: website/forms/integrations/active-campaign.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+---
+id: active-campaign
+title: Active Campaign
```
Comment (goranalkovic-infinum) [inline website/forms/integrations/active-campaign.md:3]:
```suggestion
title: ActiveCampaign
```

File: website/forms/features/success-redirect.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+---
+id: success-redirect
+title: Success redirect
```
Comment (goranalkovic-infinum) [inline website/forms/features/success-redirect.md:3]:
```suggestion
title: Success redirects
```

File: website/forms/features/multistep-forms.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
+id: multistep-forms
+title: Multistep forms
```
Comment (goranalkovic-infinum) [inline website/forms/features/multistep-forms.md:3]:
```suggestion
title: Multi-step forms
```

File: website/forms/features/multiflow-forms.md [hunk 11]
```diff
@@ -0,0 +1,6 @@
+---
+id: multiflow-forms
+title: Multiflow forms
```
Comment (goranalkovic-infinum) [inline website/forms/features/multiflow-forms.md:3]:
```suggestion
title: Multi-flow forms
```

File: website/forms/features/fallback-emails.md [hunk 12]
```diff
@@ -0,0 +1,6 @@
+---
+id: fallback-emails
+title: Fallback emails
```
Comment (goranalkovic-infinum) [inline website/forms/features/fallback-emails.md:3]:
```suggestion
title: Fallback e-mails
```

File: website/forms/features/debug.md [hunk 13]
```diff
@@ -0,0 +1,6 @@
+---
+id: debug
+title: Debug
```
Comment (goranalkovic-infinum) [inline website/forms/features/debug.md:3]:
```suggestion
title: Debugging
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by piqusy — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

File: website/docs/additional-libraries/storybook.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
+---
+id: storybook
+title: Eightshift Storybook
+---
+
+[![GitHub tag](https://img.shields.io/github/tag/infinum/eightshift-storybook.svg?style=for-the-badge)](https://github.com/infinum/eightshift-storybook)
+
+This library hosts all the packages necessary for loading Storybook in your project. We are making updates as a new core version come out.
```
Comment (piqusy) [inline website/docs/additional-libraries/storybook.md:8]:
```suggestion
This library hosts all the packages necessary for loading Storybook in your project. We are making updates as new core versions are released.
```

**Review by piqusy — CHANGES_REQUESTED** [review]
Sorry, I've made a nonsensical wording suggestion previously.
Please consider the new suggestion instead

**Pull request merged**